components/tcf/firstLayer/src/index.js

```diff
@@ -105,23 +102,14 @@ export default function TcfFirstLayer({
   }
 
   const Content = ({modalType}) => {
```

Comment:
You should remove the param modalType because is not used any more

Reply:
@maryammdot Done 👍